tests/test_vm_layer_warning.c

```
@@ -220,16 +212,23 @@ static int test_jail_openat_inside_jail(void) {
 
-    int fd = fs_jail_openat(tmp_path, O_RDONLY, 0);
+    char tmpl[] = "fl_jail_test_XXXXXX";
+    int tmp_fd = mkstemp(tmpl);
+    if (tmp_fd < 0) {
+        fprintf(stderr, "SKIP: mkstemp in jail cwd failed\n");
+        reset_jail_state();
+        return 0;
+    }
+    close(tmp_fd);
+
+    int fd = fs_jail_openat(tmpl, O_RDONLY, 0);
```

Clang (14.0.6)

[warning] variable 'fd' is not initialized

(cppcoreguidelines-init-variables)